src/components/Heading/Heading.tsx

```diff
@@ -17,6 +17,16 @@ export const VARIANTS = [
 ] as const;
 export type Variant = typeof VARIANTS[number];
 export type HeadingElement = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+// For now, "h1"-"h6" sizes point to the old type ramp, while
+// design token sizes point to the new type ramp.
+// These will be brought in sync with the next major release.
```

Comment:
Here's a ticket for how we'll bring them in sync: https://app.shortcut.com/czi-edu/story/196525/test-out-new-type-ramp-on-platform (I will make this ticket better before then)

Essentially, we're going to fully switch to the new type ramp and codemod traject headings to maintain their smaller size (size "h1" becomes "h3" in the new ramp, etc). However, as we merge in changes from `next`, I want to make sure it's clear to us within the EDS repo which size we want. I'm thinking EDS will internally use the design token names, but we'll still let app developers use "h1" - "h6" for convenience (this will be well documented)

Reply:
Could you specify in the comment that when you say "design token", you're referring to the "headline-" and "title-" tokens? This has wording has been throwing me off a little bit because I think of the "h[size]" ones as tokens too, and it's not always clear to me what the "design token" is.

Reply:
ohh yes, thanks!